dnsforward/ipset.go

```diff
@@ -89,10 +89,49 @@ func (c *ipsetCtx) getIP(rr dns.RR) net.IP {
 	}
 }
 
-// Add IP addresses of the specified in configuration domain names to an ipset list
-func (c *ipsetCtx) process(ctx *dnsContext) int {
+// Find the ipsets for a given host (accounting for subdomain wildcards)
```

Comment:
It's better to start doc comments for unexported identifiers just like one would for exported, that is:

```go
// getIpsetNames finds the ipstes […]
```

That applies to the other doc comments below.